### PR TITLE
Increase the default max buckets

### LINKED ISSUE #2765: Internal error Max buckets exceeded
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.0.3 | 7.10.0 | 4014 |

**Description**
An error was detected when entering Statistics and executing a query greater than 1 day (today). It generates the following error:
`error: "Internal Server Error"
message: "Error: Max buckets exceeded: 18000 of 2000 allowed. Choose a larger interval or a shorter time span"
statusCode: 500`

**Steps to reproduce**

1. Go to 'Wazuh Menu --> Management --> Statistics '
2. Click on 'Filters ex: _**Last year**_'
3. The view does not show data 
4. See an error on a network of browsers.

**Solution**
Kibana configuration:
Kibana -> Stack Management -> Advanced Settings -> timelion -> **max_buckets** (default 2000) increase to 20000

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team! This PR solved the problem that was caused when entering Statistics and executing a query greater than 1 day. 
Closes #2765

### CODE REVIEW

File: public/components/health-check/lib/check-max-buckets.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    return false;
+  }
+
+  const maxBuckets = settings["timelion:max_buckets"].userValue;
+  const maxBucketsObject = JSON.parse(maxBuckets);
+  return WAZUH_MAX_BUCKETS_DEFAULT === maxBucketsObject;
+}
+
+async function updateMaxBucketsSetting(isModified: boolean) {
+  console.log("ENTRA AQUI 3")
```
Comment (Desvelao) [inline public/components/health-check/lib/check-max-buckets.ts:39]: Please remove this `console.log`

File: public/components/health-check/lib/check-max-buckets.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  return !isModified && await GenericRequest.request(
+    'POST',
+    '/api/kibana/settings',
+    { "changes": { "timelion:max_buckets": JSON.stringify(WAZUH_MAX_BUCKETS_DEFAULT) } }
```
Comment (Desvelao) [inline public/components/health-check/lib/check-max-buckets.ts:43]: The `WAZUH_MAX_BUCKETS_DEFAULT` is a number, you don't need to do:

```
JSON.stringify(WAZUH_MAX_BUCKETS_DEFAULT)
```

File: public/components/health-check/lib/check-max-buckets.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  return kibanaSettings.data;
+}
+
+async function checkMaxBuckets({ settings }: responseKbnSettings) {
+  if (!settings["timelion:max_buckets"]) {
+    return false;
+  }
+
+  const maxBuckets = settings["timelion:max_buckets"].userValue;
+  const maxBucketsObject = JSON.parse(maxBuckets);
```
Comment (Desvelao) [inline public/components/health-check/lib/check-max-buckets.ts:34]: Why use `JSON.parse` here? It should be a number.

**Review by frankeros — CHANGES_REQUESTED** [review]
I tested it, and everything seems ok, but you forgot to add the config in `initial-wazuh-congit.ts`, please add it and add the changelog.

Good Job!

**Pull request merged**